app/src/main/res/values-sat/strings.xml

```diff
@@ -238,9 +238,6 @@
     <!-- Search engine suggestion title text. The first parameter is the name of teh suggested engine-->
     <string name="search_engine_suggestions_title">%s ᱥᱮᱸᱫᱽᱨᱟᱭ ᱢᱮ</string>
 
-    <!-- Search engine suggestion description text -->
-    <string name="search_engine_suggestions_description">ᱴᱷᱤᱬᱟ ᱵᱟᱨ ᱠᱷᱚᱱ ᱥᱤᱫᱷᱟᱹ ᱥᱮᱸᱫᱽᱨᱟᱭ ᱢᱮ</string>
```

Comment:
This string is [still in use](https://github.com/mozilla-mobile/fenix/blob/6034adcbc8576c0fbbfc201464f360c3b4bafa47/app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt#L200) @Delphine, is there any reason why we are deleting it for this locale?

Reply:
Hmmm that's strange indeed. I have no idea why this would be the case 🤔

Reply:
No problem, I remove the commit https://github.com/mozilla-mobile/fenix/pull/20878#issuecomment-909381624 until to unblock this from landing